content/agent/proxy.md

```diff
@@ -29,22 +29,66 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
 
-##### Agent v6
+### Agent v6
+
+With Agent v6 you can set different proxy servers for `https` and `http` requests in your `datadog.yaml` file. The Agent always uses `https` to send data to Datadog, but integrations might use `http` to collect metrics from different software.
```

Comment:
With Agent v6, you can set different proxy servers for `https` and `http` requests in `datadog.yaml`. The Agent uses `https` to send data to Datadog, but integrations might use `http`.